protobuf.bzl

```diff
@@ -141,10 +142,12 @@ def cc_proto_library(
         deps=[s + "_genproto" for s in deps],
         includes=includes,
         protoc=protoc,
+        visibility=visibility,
```

Comment:
or we could use visibility for the cc_library and add additional:
proto_visibility parameter which we'd propagate to the _proto_gen rules

would that work?